microraiden/microraiden/test/fixtures/channel_manager.py

```diff
@@ -39,8 +34,12 @@ def fail(greenlet):
 @pytest.fixture
 def channel_manager(web3, receiver_privkey, make_channel_manager_proxy, token_contract, use_tester,
                     mine_sync_event):
+    if use_tester:
+        snapshot_data = web3.testing.snapshot()
```

Comment:
would rename `snapshot_data` to `snapshot_id` (it's not the actual snapshot, but only a reference)

Reply:
Yes you are right, I've got it mistaken for another function that returns binary-serialized block as a snapshot.